modules/core/localizer.js

```diff
@@ -391,5 +391,36 @@ export function coreLocalizer() {
         return code;  // if not found, use the code
     };
 
+    localizer.floatFormatter = (locale) => {
+        if (!('Intl' in window && 'NumberFormat' in Intl &&
+              'formatToParts' in Intl.NumberFormat.prototype)) {
+            return (number) => number.toString();
+        } else {
+            return (number) => number.toLocaleString(locale);
```

Comment:
`Number.prototype.toLocaleString()` and `Intl.NumberFormat()` apparently default to a maximum of three decimal digits, which means this round-tripping would actually cause rounding. We can work around that behavior by setting the `maximumFractionDigits` option to 20 (the maximum value that doesn’t result in a `RangeError`).

In practice, most occurrences of the ten-thousandths place in tag values would be from poorly executed imports, and any manual editing of those values would be unlikely to require so much precision. (This code should only run due to a user-initiated edit of the value using the field, not mere viewing or manual editing of the raw value.)

Reply:
> We can work around that behavior by setting the `maximumFractionDigits` option to 20 (the maximum value that doesn’t result in a `RangeError`).

I implemented this workaround.